src/components/ButtonGroup/__stories__/buttonGroup.stories.js

```diff
@@ -74,8 +74,51 @@ export const Sandbox = () => (
   </div>
 );
 
+export const ButtonGroupWithTooltip = () => (
+  <div
+    style={{
+      width: "380px",
+      height: "400px",
+      border: "1px solid",
```

Comment:
please add out colors to it - `var(--ui-border-color)` so it will look well in dark,black modes